docs/en/admins/02_Prerequisites.md

```diff
@@ -10,7 +10,7 @@ You need to verify that your server can run FreshRSS before installing it. If yo
 | PHP           | **PHP 7+**              |                         |
 | PHP modules   | Required: libxml, cURL, JSON, PDO_MySQL, PCRE and ctype.<br />Required (32-bit only): GMP <br />Recommended: Zlib, mbstring, iconv, ZipArchive<br />*For the whole modules list see [Dockerfile](https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/Dockerfile-Alpine#L7-L9)* | |
 | Database      | **MySQL 5.5.3+**        | SQLite 3.7.4+, PostgreSQL 9.5+          |
-| Browser       | **Firefox**             | Chrome, Opera, Safari, or Edge          |
```

Comment:
While I agree the utility of this line is debatable, I'd prefer not to see an unrelated change like this in a PR. ;-)

Reply:
oups, didn't realized it got removed, sorry

Reply:
fixed